### PR TITLE
RAC-971: Front: Add search input in process tracker

### PR DESCRIPTION
**Pull request by StevenVAIDIE**
**Description (for Contributor and Core Developer)**

In this PR i added the possibility to search job execution
Screenshot from 2021-11-16 14-54-19

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.tsx [hunk 1]
```diff
@@ -32,8 +28,13 @@ const JobExecutionList = () => {
     setJobExecutionFilter(jobExecutionFilter => ({...jobExecutionFilter, page: 1, type}));
   };
 
-  /* istanbul ignore next TODO RAC-938 */
-  const handleSearchChange = () => {};
+  const handleSearchChange = useCallback((search: string) => {
+    setJobExecutionFilter(jobExecutionFilter => ({...jobExecutionFilter, page: 1, search}));
+  }, [setJobExecutionFilter]);
```
Comment (ValentinMumble) [inline src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.tsx:33]:
```suggestion
  }, []);
```

**Review by ValentinMumble — APPROVED** [review]

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/components/JobExecutionList/JobExecutionSearchBar.test.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {screen, act} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {JobExecutionSearchBar} from "./JobExecutionSearchBar";
```
Comment (ValentinMumble) [inline src/Akeneo/Platform/Job/front/process-tracker/src/feature/components/JobExecutionList/JobExecutionSearchBar.test.tsx:4]: Lint issue I believe

**Review by ValentinMumble — APPROVED** [review]

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/components/JobExecutionList/JobExecutionSearchBar.test.tsx (hunk 2 above)
Comment (juliensnz) [reply src/Akeneo/Platform/Job/front/process-tracker/src/feature/components/JobExecutionList/JobExecutionSearchBar.test.tsx:4]: WOW

**Review by ValentinMumble — APPROVED** [review]

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/components/JobExecutionList/JobExecutionSearchBar.test.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import {getDefaultJobExecutionFilter, JobStatus} from "../../models";
```
Comment (juliensnz) [inline src/Akeneo/Platform/Job/front/process-tracker/src/feature/components/JobExecutionList/JobExecutionSearchBar.test.tsx:5]: ANd here? cc @StevenVAIDIE

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/components/JobExecutionList/JobExecutionSearchBar.test.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+import {renderWithProviders} from "@akeneo-pim-community/shared";
```
Comment (juliensnz) [inline src/Akeneo/Platform/Job/front/process-tracker/src/feature/components/JobExecutionList/JobExecutionSearchBar.test.tsx:6]: Tssss

**Review by pierrejolly — APPROVED** [review]

**Pull request merged**